mitodl/mit-open
# Add learner testimonials homepage UI

### What are the relevant tickets?

Closes mitodl/hq#4253

### Description (What does it do?)

Adds learner testimonials to the homepage.

### Screenshots (if appropriate):

- Desktop screenshots

![Screenshot 2024-05-31 at 2 47 34 PM](https://github.com/mitodl/mit-open/assets/945611/49735636-4d0d-4724-be70-b1b29fddfaa9)

- Mobile width screenshots

![Screenshot 2024-05-31 at 2 48 01 PM](https://github.com/mitodl/mit-open/assets/945611/748533ad-db15-493f-bc9c-2d705efe39fb)

### How can this be tested?

Add some testimonials to the system. The content doesn't matter so much, but be aware that there's only so much room on the cards (especially in mobile mode) so you should keep the quote to one or two short sentences. If you set a publish date for the testimonial, you may not see it on the homepage.

View the homepage. You should see the testimonials appear on the page and you should be able to page through them. This should work in desktop or mobile views. 

### Additional Context

Testimonials also appear in the channel page but there will be a separate PR for that.
